build.assets/Makefile

```diff
@@ -214,23 +214,26 @@ CONNECT_VERSION := $(BUILDBOX_VERSION)-dev
 endif
 
 #
-# Builds a Docker buildbox for Linux Connect builds
+# Builds a Docker buildbox for builds that depend on Node.js.
+# This can be used for building webassets as well as Teleport Connect.
 #
-.PHONY:buildbox-connect
-buildbox-connect:
-	if [[ $${DRONE} == "true" ]] && ! docker inspect --type=image $(BUILDBOX_CONNECT) 2>&1 >/dev/null; then docker pull $(BUILDBOX_CONNECT) || true; fi; \
+.PHONY:buildbox-node
+buildbox-node:
+	if [[ $${DRONE} == "true" ]] && ! docker inspect --type=image $(BUILDBOX_NODE) 2>&1 >/dev/null; then docker pull $(BUILDBOX_NODE) || true; fi; \
```

Comment:
I missed changing this in a recent PR. If you look at the other buildboxes, they start with something like

    $(call PULL_ON_CI,$(BUILDBOX_ARM))

Could you git rid of this line and change it to

    $(call PULL_ON_CI,$(BUILDBOX_NODE))

Reply:
f2993a1fb606bf9de154b49e93f98dd4513a11c1